include/lsst/gauss2d/evaluate.h

```diff
@@ -867,11 +867,12 @@ class GaussianEvaluator : public Object {
             }
         }
     }
-    ~GaussianEvaluator() {};
+    ~GaussianEvaluator() override {};
 
-    const Data& IMAGE_NULL_CONST() const { return this->IMAGE_NULL(); };
-    const Indices& INDICES_NULL_CONST() const { return this->INDICES_NULL(); };
-    const ImageArray<T, Data>& IMAGEARRAY_NULL_CONST() const { return this->IMAGEARRAY_NULL(); };
+    // TODO: Return to this>IMAGE_NULL(), etc. when DM-45445 is fixed
+    const Data& IMAGE_NULL_CONST() const { return this->_data_null_const; }
```

Comment:
The developer guide prefers:
```suggestion
    Data const& IMAGE_NULL_CONST() const { return this->_data_null_const; }
```